<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>628</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#000; color:#fff</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lbFileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>761</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#888; color:#fff</string>
    </property>
   </widget>
   <widget class="QLabel" name="labVersion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
    <property name="text">
     <string>SMU BootLoader vx.x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartVMC">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#656565; color:#000; border-radius:10px;</string>
    </property>
    <property name="text">
     <string>START VMC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateSMU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#656565; color:#000; border-radius:10px;</string>
    </property>
    <property name="text">
     <string>UPDATE SMU</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logPanel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#888; color:#fff</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCloseErrorLog">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#656565; color:#000; border-radius:10px;</string>
    </property>
    <property name="text">
     <string>CLOSE
ERROR LOG</string>
    </property>
   </widget>
   <widget class="QLabel" name="labIntro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select a file from the list then click on UPDATE SMU.&lt;/p&gt;&lt;p&gt;When ready, click START VMC&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labError">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>761</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#8f8; color:#000</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
